base/windows/CharsStatusWindow.js

```diff
@@ -157,7 +157,7 @@ export class CharsStatusWindow {
             let char = party_data.members[i];
             current_chars.push(char.key_name);
             let info_sprite = this.info_sprites[char.key_name];
-            info_sprite.group.visible = true ? true : false;
+            info_sprite.group.visible = true ? true : !info_sprite.group.visible;
```

Comment:
What do you want to check here? An equivalent code this line would be:
```javascript
if (true) {
  info_sprite.group.visible = true; //always going to this one.
} else {
  info_sprite.group.visible = !info_sprite.group.visible;
}
```

Reply:
I suppose if I want to force it to be true I can just make it true regardless...
`info_sprite.group.visible = true`

Reply:
I think yes. Also, I think the `info_sprite` has a `visible` property that perhaps is not necessary anymore.

Reply:
I can't find `info_sprite.visible` anywhere else, the one I changed was the only I one I think. And thanks for clearing up that little confusion earlier :P I'm just dumb sometimes